static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -33,49 +40,138 @@ export function hasContext(
+    method: 'GET',
+    query: {
+      collapse: 'release',
+      expand: 'inbox',
+    },
+  });
+
+  return promise;
+}
+
 type Props = {
   column: TableColumn<keyof TableDataRow>;
   dataRow: TableDataRow;
 };
 
 export default function QuickContext(props: Props) {
   // Will add setter for error.
-  const [error] = useState<boolean>(false);
+  const api = useApi();
+  const [error, setError] = useState<boolean>(false);
   const [loading, setLoading] = useState<boolean>(true);
-  const [data, setData] = useState<{msg: string} | null>(null);
+  const [data, setData] = useState<IssueData | null>(null);
+
+  // NOTE: Will extend when we add more type of contexts.
 
   useEffect(() => {
-    const mockData = {msg: t('Displaying Context for issue.')};
+    // Track mounted state so we dont call setState on unmounted components
+    let unmounted = false;
+
+    fetchData(api, props.dataRow, props.column)
+      .then(response => {
+        if (unmounted) {
+          return;
+        }
 
-    // Will replace with issue data.
-    const timer = setTimeout(() => {
-      setData(mockData);
-      setLoading(false);
-    }, 500);
+        setData(response);
+      })
+      .catch(() => {
+        setError(true);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
 
     return () => {
-      clearTimeout(timer);
+      // If component has unmounted, dont set state
+      unmounted = true;
     };
-  }, []);
+  }, [api, props.dataRow, props.column]);
 
   return (
-    <ContextContainer>
+    <Wrapper>
       {loading ? (
         <NoContextWrapper>
           <LoadingIndicator hideMessage size={32} />
         </NoContextWrapper>
       ) : error ? (
         <NoContextWrapper>{t('Failed to load context for column.')}</NoContextWrapper>
       ) : isIssueContext(props.dataRow, props.column) && data ? (
-        <NoContextWrapper>{data.msg}</NoContextWrapper>
+        <IssueContext data={data} />
       ) : (
         <NoContextWrapper>{t('There is no context available.')}</NoContextWrapper>
       )}
+    </Wrapper>
+  );
+}
+
+// Only includes issue status context for now.
+function IssueContext(props: {data: IssueData}) {
+  const statusTitle = t('Issue Status');
+  const {status} = props.data;
+
+  return (
+    <ContextContainer>
+      <ContextTitle>{statusTitle}</ContextTitle>
+      <ContextBody>
+        {status === 'ignored' ? (
+          <IconMute data-test-id="quick-context-mute-icon" color="gray500" size="sm" />
+        ) : status === 'resolved' ? (
+          <IconCheckmark color="gray500" size="sm" />
+        ) : (
+          <IconNot data-test-id="quick-context-not-icon" color="gray500" size="sm" />
+        )}
+        <StatusText>{status}</StatusText>
+      </ContextBody>
     </ContextContainer>
   );
 }
 
 const ContextContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  margin: ${space(1.5)};
+
+  & + & {
+    margin-top: ${space(1.5)};
+  }
```

Comment:
Does this do anything? I'm not too familiar with `& + &` tbh but the default styling has `margin: ${space(1.5)}` which includes `margin-top`, and these have the same values making this redundant?

Reply:
Hey, this is just a neat way of saying that if two of these containers beside each other (vertically since flex-direction is column) then add that space between them. It just shortens the logic that, first child shud not have that margin.

Reply:
The first child does have `margin-top: ${space(1.5)}` though technically, because on line 149 you've given them all `margin: ${space(1.5)}` which applies that space to all of the sides

In this image I've stacked 3 issue contexts together and added `background: red` to ensure the `& + &` is applied as expected

<img width="260" alt="Screen Shot 2022-10-17 at 4 53 53 PM" src="https://user-images.githubusercontent.com/22846452/196281223-82041f72-c195-4b4e-8618-ad96e2f0b2da.png">

Reply:
Hey,  removed the & + & rule you were right. I didn't notice that there was already another margin setter in that block.